File: Tudo-List.Domain/Entities/TodoListItem.cs
using Tudo_List.Domain.Enums;

namespace Tudo_List.Domain.Entities
{
    public class TodoListItem
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public string? Description { get; set; }
        public Status Status { get; set; }
        public Priority Priority { get; set; }
        public DateTime CreationDate { get; set; }
        public int UserId { get; set; }
        public User User { get; set; }
    }
}

File: Tudo-List/tudo-list.client/src/app/core/models/todo-list-item/todo-list-item.ts
import { PriorityEnum } from "../../enums/priority-enum";
import { StatusEnum } from "../../enums/status-enum";

export interface TodoListItem {
    id: string;
    title: string;
    description?: string;
    status: StatusEnum;
    priority: PriorityEnum;
    creationDate: Date;
}